pom.xml

```diff
@@ -0,0 +1,678 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <packaging>jar</packaging>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.teragrep</groupId>
+    <artifactId>rsm_01</artifactId>
+    <version>${revision}${sha1}${changelist}</version>
+    <name>rsm_01</name>
+    <description>Record Schema Mapping Library for Java</description>
+    <licenses>
+        <license>
+            <name>GNU Affero General Public License v3.0</name>
+            <url>https://www.gnu.org/licenses/agpl-3.0.txt</url>
+        </license>
+    </licenses>
+    <properties>
```

Comment:
Missing developers information https://github.com/teragrep/mvn_01/blob/main/workflows/pom.xml#L31
You can use github username and <just-number-without-extras>@teragrep.com mail

Reply:
Solved in commit  79e7942bab31d31302d436da42c0e065ce22da57